lib/perl5db.pl

```diff
@@ -532,7 +532,7 @@ BEGIN
 use vars qw($VERSION $header);
 
 # bump to X.XX in blead, only use X.XX_XX in maint
-$VERSION = '1.74';
+$VERSION = '1.75';
```

Comment:
Bumped version per @jkeenan suggestion to pass the [`cmp-version.t`](https://github.com/Perl/perl5/blob/blead/t/porting/cmp_version.t) test.